### ISSUE TITLE
Split Button -Accessibility - FastPass Issues -Nested interactive controls are not announced by screen readers

### ISSUE THREAD

**Issue by vasanthkumar258** [microsoft/fluentui#18785]
### Environment Information

- **Package version(s)**: Latest version
- **Browser and OS versions**: Chrome

### Describe the issue:

Run accessibility Fastpass , issue with Split button https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=msftAI

### Please provide a reproduction of the issue in a codepen:
This can reproduced with split button variants examples at
https://fabricweb.z5.web.core.windows.net/oufr/8.21.1/#/examples/button#Variants

#### Actual behavior:

Accessibility Insights - FastPass issue

#### Expected behavior:

Should not get any FastPass issue

### Documentation describing expected behavior

https://dequeuniversity.com/rules/axe/4.2/nested-interactive?application=msftAI
****

**Comment by gouttierre** [comment]
@vasanthkumar258 - Thank you for filing this issue and sharing the repro information. 

@smhigley - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by smhigley** [comment]
This is a known automated issue that will be solved in vNext. There are a number of problems with changing the current structure in v7/v8 that limit us from solving this issue there:
- Keyboard focus moves to the outer button, so that element needs to have interactive semantics. Space activates the button, and alt + down opens the menu.
- The inner menu button needs to remain a button so non-keyboard AT like touch screen readers or voice control can expose it
- We can't change the pattern to use two separate tab stops instead of one without breaking changes to DOM and interaction

That said, we've done our best to mitigate the practical effects of the current structure. If you (or anyone else) finds a problem with the screen reader (or other AT) experience with the button, we'll look into it.

**Comment by vasanthkumar258** [comment]
@smhigley, @gouttierre  , When can we expect vNext version to be released?
